src/SprykerEco/Zed/Unzer/Business/UnzerBusinessFactory.php

```diff
@@ -490,7 +503,7 @@ public function createUnzerPaymentMapper(): UnzerPaymentMapperInterface
      */
     public function createPaymentProcessorResolver(): UnzerPaymentProcessorResolverInterface
     {
-        $unzerPaymentProcessorsCollection = [
+        return new UnzerPaymentProcessorResolver([
```

Comment:
please also extract this array to a separate method

Reply:
Fixed.